### PR TITLE
feat: add logs command

### PR DESCRIPTION
**Pull request by PatrickMenoti**
**WHAT**
- Add logs cells command;

### CODE REVIEW

File: messages/logs/cells/messages.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package cells
+
+var (
+	Usage            = "cells"
+	ShortDescription = "Displays cells console logs"
+	LongDescription  = "Displays cells console logs"
+	FlagHelp         = "Displays more information about the logs cells command"
+	FlagPretty       = "Displays logs in a prettified way"
+	FlagFunctionId   = "ID of the function you wish the see the logs for; if not informed, logs for all functions will be displayed"
```
Comment (lmalheiro) [inline messages/logs/cells/messages.go:9]:
```suggestion
	FlagFunctionId   = "ID of the function for which you wish to see the logs; if not provided, logs for all functions will be displayed."
```

Comment (PatrickMenoti) [reply messages/logs/cells/messages.go:9]: @gabriel-azion will create another PR after this, with the final messages... these ones are just placeholders :)

Comment (jotanarciso) [reply messages/logs/cells/messages.go:9]: ok.

**Review by jotanarciso — APPROVED** [review]

File: pkg/api/graphql/cells/cells.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	"time"
+
+	msg "github.com/aziontech/azion-cli/messages/logs/cells"
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/machinebox/graphql"
+)
+
+type CellsConsoleEvent struct {
+	Ts              time.Time `json:"ts"`
+	SolutionId      string    `json:"solutionId"`
```
Comment (maxwelbm) [inline pkg/api/graphql/cells/cells.go:15]: Does JSON come in this kind of formatting?  CamelCase?

Comment (PatrickMenoti) [reply pkg/api/graphql/cells/cells.go:15]:
```
{
        "ts": "2024-01-30T20:11:00Z",
        "solutionId": "1531930033",
        "configurationId": "1706643363",
        "functionId": "16147",
        "id": "c89728ff-7261-4cb8-bf60-d219d0668975",
        "lineSource": "CONSOLE",
        "level": "LOG",
        "line": "Worker responding with: Hello World"
      }
```

File: pkg/cmd/logs/cells/cells.go [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+	cmd.Flags().BoolP("help", "h", false, msg.FlagHelp)
+	cmd.Flags().StringVar(&functionId, "function-id", "", msg.FlagFunctionId)
+	cmd.Flags().StringVar(&limit, "limit", "100", msg.LimitFlag)
+	cmd.Flags().BoolVar(&tail, "tail", false, msg.FlagTail)
+	cmd.Flags().BoolVar(&pretty, "pretty", false, msg.FlagPretty)
+	return cmd
+}
+
+func printLogs(cmd *cobra.Command, f *cmdutil.Factory) error {
```
Comment (maxwelbm) [inline pkg/cmd/logs/cells/cells.go:55]: Top man 🚀

**Review by maxwelbm — APPROVED** [review]

File: pkg/api/graphql/cells/cells.go [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+	formattedTime := currentTime.Format("2006-01-02T15:04:05")
+
+	filter := ""
+	if functionId != "" {
+		filter = fmt.Sprintf(`functionId: "%s"`, functionId)
+	}
+
+	limit := "limit: " + limitFlag
+
+	query := `
```
[the remark this answers is not in the document]
Comment (PatrickMenoti) [reply pkg/api/graphql/cells/cells.go:40]: It's a good idea. I will use this when creating the next log command :D

**Comment by PatrickMenoti** [comment]
> LGTM. No tests though... ;-)

Since I'll be working on the next log command I could add tests to both :D should be fairly easy

**Pull request merged**